sqlstore/src/main/resources/db/migration_afterMigrate/70-kits.sql

```diff
@@ -0,0 +1,3 @@
+-- StartNoTest
```

Comment:
FYI, you've got a dash after the number here, and an underscore after the number elsewhere.